index.php

```diff
@@ -0,0 +1,204 @@
+<?php
+require_once(dirname(__FILE__) . '/includes/theme-compat/header.php');
+require_once($_SERVER['DOCUMENT_ROOT'] . '/includes/db-connect.php');
```

Comment:
One time you use `dirname` then right below you use `$_SERVER['DOCUMENT_ROOT']`. Is there a reason for that?

Reply:
Fixed